aurblobs/repository.py

```diff
@@ -260,7 +307,7 @@ def sign_and_add(self, pkgroot):
         for line in container.logs(stream=True):
             print('\t{0}'.format(line.decode().rstrip('\n')))
 
-        return container.wait() == 0
+        return container.wait()['StatusCode'] == 0
```

Comment:
require `docker>=3.0`

Reply:
Good catch. This pattern was already used before in `package.py`. Therefore i will update the requirement...

Reply:
yup, my mistake :+1: